backend/src/test/java/com/pickpick/message/application/MessageServiceQueryTest.java

```diff
@@ -0,0 +1,460 @@
+        List<Message> freeChatMessages = createAndSaveMessages(freeChat, summer);
+        List<Message> allMessages = messages.findAll();
+
+        ChannelSubscription first = subscriptions.save(new ChannelSubscription(notice, summer, VIEW_ORDER_FIRST));
+        ChannelSubscription second = subscriptions.save(new ChannelSubscription(freeChat, summer, VIEW_ORDER_SECOND));
+
+        private List<Message> createAndSaveMessages(final Channel channel, final Member member) {
+            List<Message> messagesInChannel = Arrays.stream(MessageFixtures.values())
+                    .map(messageFixture -> messageFixture.create(channel, member))
+                    .collect(Collectors.toList());
+
+            for (Message fixture : messagesInChannel) {
+                messages.save(fixture);
+            }
+
+            return messagesInChannel;
+        }
+
+        @Nested
+        @DisplayName("조회 조건과 관련없이")
+        class alwaysFilter {
+
+            MessageRequest request = fromLatestInChannels(List.of(notice), allMessages.size() + 1);
+            MessageResponses response = messageService.find(summer.getId(), request);
+
+            @DisplayName("텍스트가 비어있는 메시지는 제외된다")
+            @Test
+            void filterEmptyMessages() {
+                List<MessageResponse> foundMessages = response.getMessages();
+                boolean isEmptyMessageFiltered = foundMessages.stream()
+                        .noneMatch(message -> message.getText().isEmpty());
+
+                assertThat(isEmptyMessageFiltered).isTrue();
+            }
+        }
+
+        @DisplayName("더 이상 조회할 메시지가 없다면")
+        @Nested
+        class noMoreMessagesRemain {
+
+            MessageRequest request = onlyCount(noticeMessages.size());
+            MessageResponses response = messageService.find(summer.getId(), request);
+
+            @DisplayName("isLast는 true이다")
+            @Test
+            void messageRemainIsLastFalse() {
+
+                boolean isLast = response.isLast();
+
+                assertThat(isLast).isTrue();
+            }
+        }
+
+        @DisplayName("조회 할 메시지가 남아있다면")
+        @Nested
+        class messagesRemain {
+
+            MessageRequest request = onlyCount(MESSAGE_COUNT);
+            MessageResponses response = messageService.find(summer.getId(), request);
+
+            @DisplayName("isLast는 false다")
+            @Test
+            void messageRemainIsLastTrue() {
+
+                boolean isLast = response.isLast();
+
+                assertThat(isLast).isFalse();
+            }
+        }
+
+        @DisplayName("특정 단어로 여러 채널에서 검색한다면")
+        @Nested
+        class keywordInChannels {
+
+            String keyword = "줍줍";
+
+            MessageRequest request = searchByKeywordInChannels(List.of(notice, freeChat), keyword, allMessages.size());
+            MessageResponses response = messageService.find(summer.getId(), request);
+
+            @DisplayName("해당 채널에 있고, 특정 단어가 포함된 메시지가 조회된다")
```

Comment:
DisplayName을 합쳐보면 `특정 단어로 여러 채널에서 검색한다면 해당 채널에 있고, 특정 단어가 포함된 메시지가 조회된다` 라는 말이 완성되는데요
```suggestion
            @DisplayName("해당 채널들에서 특정 단어가 포함된 메시지가 조회된다")
```
위와 같은 변경은 어떨까 제안드려 봅니다👀